src/sentry/services/hybrid_cloud/organization/service.py

```diff
@@ -76,6 +76,59 @@ def get_organization_by_id(
         """
         pass
 
+    @regional_rpc_method(
+        resolve=ByOrganizationId("resolving_org_id"), return_none_if_mapping_not_found=True
+    )
```

Comment:
I'm rather pleased with this part. I think it solves one of the problems that stopped us from doing this before, was that we would need to do several manual lookups on `OrganizationMapping` to split the set of org IDs into regional batches, and we hadn't found an elegant way to work that into the service interface.

What's nice here is that we look up only the one `OrganizationMapping` (implicitly, as with all `regional_rpc_method` calls) and offload the work of looking up the rest of them to whichever region silo we arbitrarily picked to go to first. Whatever we get back from that region, `get_organizations_in_batches` deletes from its to-do list, then repeats until the to-do list is empty.